### ISSUE TITLE
Document how to release a new Scala Steward version

### ISSUE THREAD

**Issue by fthomas** [scala-steward-org/scala-steward#2486]
This is what I currently do to publish a new Scala Steward version:
1. run `./scripts/release.sh x.y.z` where `x.y.z` is the new version
1. check that the CI publish job of the newly pushed tag ran successfully
1. publish the latest release notes draft at https://github.com/scala-steward-org/scala-steward/releases with the new version
1. close the milestone with the old version and create a new milestone with the new version at https://github.com/scala-steward-org/scala-steward/milestones

These steps should be documented somewhere.

**Comment by henricook** [comment]
Do you have a tool for creating a release notes draft @fthomas ? Or do you keep track of all the merge requests and add them one at a time, as they get merged?

**Comment by fthomas** [comment]
> Do you have a tool for creating a release notes draft @fthomas ?

Yes. https://github.com/apps/release-drafter creates the release notes automatically. The only thing I'm doing manually is to label PRs so that Release Drafter puts them into the right category and to add a milestone so that they are associated with the correct version.
